modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/MockedStructuresTest.java

```diff
@@ -194,13 +199,23 @@ void before() throws NodeStoppingException {
             });
         };
 
-        dsm = new DataStorageManager(configRegistry, workDir) {
-            /** {@inheritDoc} */
-            @Override
-            protected Iterable<StorageEngineFactory> engineFactories() {
-                return List.of((configurationRegistry, storePath) -> new RocksDbStorageEngine(rocksDbEngineConfig, storePath));
-            }
-        };
+        dsm = new DataStorageManager(
+                configRegistry,
+                workDir,
+                List.of(new StorageEngineFactory() {
```

Comment:
Can't you instantiate "RocksDbStorageEngineFactory" instead of making anonymous class?
I believe it would be easy to mock necessary ConfigurationRegistry methods to provide rocksdb configuration, right?

Reply:
Fix it